src/lib/analytics.js

```diff
@@ -34,10 +35,14 @@ class Analytics {
 
     if ( !this.id ) return null;
 
-    const config = this.optimize_id ? {
+    let config = this.optimize_id ? {
       optimize_id: this.optimize_id
     } : {};
 
+    for(let i = 0; i < Object.keys(this.config).length; i++) {
```

Comment:
what's your thoughts on making this block along the lines of:
```
Object.keys(this.config).forEach(key => {
  config[key] = this.config[key];
})
```
Might be a little easier to read and you don't have to use `Object.keys(this.config)` multiple times. 

Alternatively, we could probably even just spread it onto the original object since you're using a `let` now:
```
config = {
  ...config,
  ...this.config
}
```

Reply:
I really liked your second suggestion and put it in. 
It's compact and easy to read/understand.
Thanks for the suggestion.